### PR TITLE
Add vcr module and vcr to all tests

### PR DESCRIPTION
**Pull request by SMJ289**
## Type of change made:
- [ ] **Bugfix** (non-breaking change which fixes an issue)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Refactoring** (improves existing functionality)
- [x] **Testing** (tests existing code or improves existing tests)
- [ ] **Styling** (no functionality changed, just visual improvements)
## Description:

Adds VCR module and generates cassettes for all tests.

## What functionality does this accomplish?

- Speeds up testing and reduce calls to third party APIs
- Steve no longer gets a tweet every time the tests are ran

closes # | does not close anything

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

**We should still try to implement unit testing**

### CODE REVIEW

**Comment by jklich151** [comment]
Added VCR to stub the api request. 
Speeds up testing.

File: test_app.py [hunk 1]
```diff
@@ -18,7 +22,9 @@ def test_it_returns_users():
 def test_it_returns_members_by_state():
     global test_browser
     state = 'co'
-    response = test_browser.get(f'/members_by_state/{state}')
+    with vcr.use_cassette('fixtures/vcr_cassettes/members_by_state.yaml'):
+      response = test_browser.get(f'/members_by_state/{state}')
```
Comment (itemniner) [inline test_app.py:26]: This is a cool way to user vcr, I wonder if there is a webmock version on this.

File: fixtures/vcr_cassettes/members_by_state.yaml [hunk 2]
```diff
@@ -0,0 +1,186 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
```
Comment (itemniner) [inline fixtures/vcr_cassettes/members_by_state.yaml:7]: I like the response, does this file get generated?

File: test_app.py (hunk 1 above)
Comment (SMJ289) [reply test_app.py:26]: Hmm I wonder. I was honestly pretty surprised to find vcr for Python. ❤️  VCR!

File: fixtures/vcr_cassettes/members_by_state.yaml (hunk 2 above)
Comment (SMJ289) [reply fixtures/vcr_cassettes/members_by_state.yaml:7]: This file was generated after adding `with vcr` before the requests. Then each test references the saved cassette afterward.

**Pull request merged**